src/devices/Bmx280/Bmx280.csproj

```diff
@@ -3,16 +3,19 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <EnableDefaultItems>false</EnableDefaultItems>
+    <RootNamespace>Iot.Device.Bmx280</RootNamespace>
```

Comment:
Why did you need to add this?

Reply:
That was to ensure that newly created/refactored files used the correct namespace. Otherwise the default, defaults to just `Bmx280` IIRC. This also makes static analysis happy (Resharper really wanted to fix that namespace).

Reply:
Should I pull it out?

Reply:
Oh ok, no it's fine to leave it. I was just wondering so that if it was useful we would add it to the binding template. Sounds like it may be good to add it there at some point.